src/DssSpell.sol

```diff
@@ -48,36 +153,321 @@ contract DssSpellAction is DssAction {
+            usds: USDS,
+            // Init USDS -> SKY rewards with sky parameter being 0x56072C95FAA701256059aa122697B133aDEd9279
+            sky: SKY,
+            // Init USDS -> SKY rewards with rewards parameter being 0x0650CAF159C5A49f711e8169D4336ECB9b950275
+            rewards: REWARDS_USDS_SKY,
+            // Init USDS -> SKY rewards with rewardsKey parameter being REWARDS_USDS_SKY
+            rewardsKey: "REWARDS_USDS_SKY",
+            // Init USDS -> SKY rewards with dist parameter being 0x2F0C88e935Db5A60DDA73b0B4EAEef55883896d9
+            dist: REWARDS_DIST_USDS_SKY,
+            // Init USDS -> SKY rewards with distKey parameter being REWARDS_DIST_USDS_SKY
+            distKey: "REWARDS_DIST_USDS_SKY",
+            // Init USDS -> SKY rewards with vest parameter being 0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9
+            vest: MCD_VEST_SKY,
+            // Init USDS -> SKY rewards with vestTot parameter being 600M * WAD
+            vestTot: 600 * MILLION * WAD,
+            // Init USDS -> SKY rewards with vestBgn parameter being block.timestamp - 7 days
+            vestBgn: block.timestamp - 7 days,
+            // Init USDS -> SKY rewards with vestTau parameter being 365 days - 1
+            vestTau: 365 days - 1
+        }));
+
+        // Call distribute() in VestedRewardsDistribution contract in the spell execution
+        VestedRewardsDistributionLike(REWARDS_DIST_USDS_SKY).distribute();
+
+        // SKY Vesting Stream  | from 'block.timestamp - 7 days' for '365 days - 1' | 600M * WAD SKY | 0x2F0C88e935Db5A60DDA73b0B4EAEef55883896d9
+
+        // Note: the instructions above are executed through UsdsSkyFarmingInit.init()
+
+        // Initialize the new cron job by calling VestedRewardsDistributionJobInit.init with the following parameters:
+        VestedRewardsDistributionJobInit.init(
+            // Initialize cron job with job parameter being 0x6464C34A02DD155dd0c630CE233DD6e21C24F9A5
+            CRON_REWARDS_DIST_JOB,
+            VestedRewardsDistributionJobInitConfig({
+                // Initialize cron job with cfg.jobKey parameter being CRON_REWARDS_DIST_JOB
+                jobKey: "CRON_REWARDS_DIST_JOB"
+            })
+        );
+
+        // Add VestedRewardsDistribution to the new cron job by calling VestedRewardsDistributionJobInit.setDist with the following parameters:
+        VestedRewardsDistributionJobInit.setDist(
+            // Add VestedRewardsDistribution to the new cron job with job parameter being 0x6464C34A02DD155dd0c630CE233DD6e21C24F9A5
+            CRON_REWARDS_DIST_JOB,
+            VestedRewardsDistributionJobSetDistConfig({
+                // Add VestedRewardsDistribution to the new cron job with cfg.dist parameter being 0x2F0C88e935Db5A60DDA73b0B4EAEef55883896d9
+                dist: REWARDS_DIST_USDS_SKY,
+                // Add VestedRewardsDistribution to the new cron job with cfg.interval parameter being 7 days
+                interval: 7 days
```

Comment:
Please update the instruction and the parameter to match [latest one](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=1165645248#gid=1165645248&range=B149)